packages/sw-lib/src/index.js

```diff
@@ -26,6 +26,33 @@ if (!assert.isSWEnv()) {
   throw ErrorFactory.createError('not-in-sw');
 }
 
+/**
+ * The sw-lib module is a high-level library that makes it easier to
+ * configure routes with caching strategies as well as manage precaching
+ * of assets during the install step of a service worker.
+ *
+ * @example
+ * importScripts('/<Path to Module>/build/sw-lib.min.js');
+ *
+ * goog.swlib.cacheRevisionedAssets([
+ *     '/styles/main.1234.css',
```

Comment:
I think what you're trying to show here is that `cacheRevisionedAssets` accepts both filename paths with revisions in there or an object with the raw path and a separate revision number. Is that the case? 

It might be worth splitting these into two examples (one showing caching a few files with one approach (e.g `/styles/main.1234.css', '/js/main.1234.js' etc) and then the object variation. Might make it more easy to read through.